configs/denver-casr.nrel-op.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://denver-casr-openpath.nrel.gov/api/",
+        "aggregate_call_auth": "user_only"
+    },
+    "intro": {
+        "program_or_study": "program",
```

Comment:
From our discussion today, I think that they may want to go with `study` instead of `program` so that they don't have to send out individual emails with tokens. I'm going to merge this for now, but we may have to revisit in a month